Can't start main UI dev server

<!--
Please DO NOT ERASE this template, but fill in the details as requested. This will ensure your issue is properly filed and will be considered. Otherwise, we may reserve the right to close it without further action.
-->

## The problem
I cloned this repo, changed to bundles/org.openhab.ui/web, ran npm install and then npm start.
But npm start fails with this output:
```

> openhab@3.0.0 start
> npm run dev


> openhab@3.0.0 dev
> cross-env NODE_ENV=development webpack-dev-server --config ./build/webpack.config.js

[HPM] Proxy created: [
  '/auth',
  '/rest',
  '/chart',
  '/proxy',
  '/icon',
  '/static',
  '/changePassword',
  '/createApiToken'
]  ->  http://localhost:8080
ℹ ｢wds｣: Project is running at http://localhost:8081/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /www/
ℹ ｢wds｣: 404s will fallback to /index.html
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
ℹ ｢wdm｣: wait until bundle finished: /
/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/chokidar.js:17
throw new Error(
^

Error: No version of chokidar is available. Tried chokidar@2 and chokidar@3.
You could try to manually install any chokidar version.
chokidar@3: Error: Cannot find module 'chokidar'
Require stack:
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/chokidar.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/DirectoryWatcher.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watcherManager.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watchpack.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/node/NodeWatchFileSystem.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/node/NodeEnvironmentPlugin.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/webpack.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack-dev-server/bin/webpack-dev-server.js
chokidar@2: Error: Cannot find module 'watchpack-chokidar2'
Require stack:
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/chokidar.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/DirectoryWatcher.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watcherManager.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watchpack.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/node/NodeWatchFileSystem.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/node/NodeEnvironmentPlugin.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/webpack.js
- /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack-dev-server/bin/webpack-dev-server.js

    at Object.<anonymous> (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/chokidar.js:17:7)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/DirectoryWatcher.js:9:16)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at WatcherManager.getDirectoryWatcher (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watcherManager.js:14:25)
    at WatcherManager.watchFile (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watcherManager.js:28:14)
    at Watchpack.<anonymous> (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watchpack.js:38:49)
    at Array.map (<anonymous>)
    at Watchpack.watch (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/watchpack/lib/watchpack.js:37:28)
    at NodeWatchFileSystem.watch (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/node/NodeWatchFileSystem.js:70:16)
    at Watching.watch (/mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/Watching.js:114:48)
    at /mnt/ssd2/Code/openhab-webui/bundles/org.openhab.ui/web/node_modules/webpack/lib/Watching.js:101:10
```

## Expected behavior

start up normally

## Steps to reproduce

<!--
Provide accurate steps to reproduce the issue, including pastes of widget/page code if necessary.
-->

1. 


## Your environment

<!--
As an admin, in the main UI, choose *Help & About* on the left sidebar, expand *Technical information* and click on *View details*, then click *Copy* and paste the results here. You may omit information that is not pertinent to this issue if you feel it's divulging information you'd like not to share.
-->

```yaml
██████████████████  ████████   alwin@alwin-pc 
██████████████████  ████████   -------------- 
██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: HP Compaq Elite 8300 MT 
████████            ████████   Kernel: 5.10.123-1-MANJARO 
████████  ████████  ████████   Uptime: 3 hours, 29 mins 
████████  ████████  ████████   Packages: 2715 (pacman), 45 (flatpak) 
████████  ████████  ████████   Shell: zsh 5.9 
████████  ████████  ████████   Resolution: 1680x1050, 1920x1080, 1680x1050 
████████  ████████  ████████   DE: Plasma 5.24.5 
████████  ████████  ████████   WM: KWin 
████████  ████████  ████████   WM Theme: ActiveAccentDusk 
████████  ████████  ████████   Theme: ReversalDark [Plasma], Adwaita-dark [GTK3] 
████████  ████████  ████████   Icons: Fluent-dark [Plasma], Fluent-dark [GTK2/3] 
                               Terminal: vscode 
                               CPU: Intel i5-3570 (4) @ 3.800GHz 
                               GPU: NVIDIA GeForce GTX 1050 
                               Memory: 8071MiB / 15950MiB 

```

## Browser console

<!--
Open the developer tools in your browser, go to the Console tab and paste errors and other messages that might be relevant to this issue.
You may also paste a screenshot if you prefer.
-->

```txt
does not apply
```

## Browser network traffic

<!--
Open the developer tools in your browser, go to the Network tab and paste
screenshots of the network traffic and the details of individual requests that failed.
-->
does not apply
## Additional information

<!--
Provide any information not pertinent to the above sections that you'd still like to share.
-->
none

--- a/bundles/org.openhab.ui/web/package.json
+++ b/bundles/org.openhab.ui/web/package.json
@@ -126,6 +126,7 @@
     "babel-plugin-dynamic-import-node-babel-7": "^2.0.7",
     "babel-plugin-transform-vue-jsx": "^4.0.1",
     "chalk": "^2.4.2",
+    "chokidar": "^3.5.3",
     "copy-webpack-plugin": "^4.6.0",
     "cross-env": "^5.2.1",
     "css-loader": "^2.1.1",

